### ISSUE TITLE
Two 'Notice Procedure' lines in comparison table when there should be only one

### ISSUE THREAD

**Issue by ostephens** [k-int/KBPlus#961]
Comparing the "Taylor & Francis_NESLi2_Taylor & Francis Journals_2015-2017" ONIX-PL licence to "JISC Collections/Dataset Sub/ICE Publishing/Institution of Civil Engineers Virtual Library Archive 1836-2001/01012007-31122013" licence.

In XML both have GeneralTerm->NoticeProcedure:

```
 <GeneralTerm>
            <GeneralTermType>onixPL:NoticeProcedure</GeneralTermType>
            <LicenseTextLink href="Ex140100"/>
            <LicenseTextLink href="Ex140101"/>
            <LicenseTextLink href="Ex140102"/>
            <GeneralTermRelatedPlace>
                <GeneralTermPlaceRelator>onixPL:AddressForNoticesToLicensor</GeneralTermPlaceRelator>
                <RelatedPlace>LicensorAddressForNotices</RelatedPlace>
            </GeneralTermRelatedPlace>
            <GeneralTermRelatedPlace>
                <GeneralTermPlaceRelator>onixPL:AddressForNoticesToLicensee</GeneralTermPlaceRelator>
                <RelatedPlace>LicenseeAddressForNotices</RelatedPlace>
            </GeneralTermRelatedPlace>
        </GeneralTerm>
```

and

```
<GeneralTerm>
            <GeneralTermType>onixPL:NoticeProcedure</GeneralTermType>
            <LicenseTextLink href="Ex140100"/>
            <LicenseTextLink href="Ex140101"/>
            <LicenseTextLink href="d1e108d1e168920100605T210207.093Z"/>
            <LicenseTextLink href="Ex140102"/>
            <GeneralTermRelatedPlace>
                <GeneralTermPlaceRelator>onixPL:AddressForNoticesToLicensor</GeneralTermPlaceRelator>
                <RelatedPlace>Sub-LicensorAddressForNotices</RelatedPlace>
            </GeneralTermRelatedPlace>
            <GeneralTermRelatedPlace>
                <GeneralTermPlaceRelator>onixPL:AddressForNoticesToLicensee</GeneralTermPlaceRelator>
                <RelatedPlace>Sub-LicenseeAddressForNotices</RelatedPlace>
            </GeneralTermRelatedPlace>
        </GeneralTerm>
```

For some reason this results in two lines in the comparison table rather than one:

<img width="1192" alt="screen shot 2015-07-15 at 15 54 30" src="https://cloud.githubusercontent.com/assets/576174/8701512/1492665e-2b0a-11e5-8479-b90b8ba95eab.png">

Can't see why as the terms look close to identical.

**Comment by iyawnis** [comment]
This is strange.. on some licences this displays correctly as one row... will keep looking...
![screenshot from 2015-07-16 10 15 40](https://cloud.githubusercontent.com/assets/2336352/8719435/a2e0777c-2ba3-11e5-9054-351a6345e65c.png)

**Comment by ostephens** [comment]
Is it because of the of the 

```
<RelatedPlace>LicensorAddressForNotices</RelatedPlace>
```

vs

```
<RelatedPlace>Sub-LicensorAddressForNotices</RelatedPlace>
```

in the licences I'm using for comparison?

**Comment by ostephens** [comment]
Another example of two rows displaying when one expected - again some differences in values within the comparison point

<img width="1202" alt="screen shot 2015-07-16 at 12 29 42" src="https://cloud.githubusercontent.com/assets/576174/8721951/64ac45a8-2bb7-11e5-9548-1e2ce7ac5a1b.png">

I'm assuming that these have the same root cause, but happy to create separate github issue as for #964 if you want

**Comment by iyawnis** [comment]
UsageMethod is causing the problem on that one, noticed it few mins ago.

**Comment by ostephens** [comment]
Any progress with this?

**Comment by iyawnis** [comment]
Started looking at this again, fixed #964
